qns/Dockerfile

```diff
@@ -10,10 +10,11 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
  && apt-get autoremove -y && apt-get clean -y \
  && rm -rf /var/lib/apt/lists/*
 
+ARG RUST_VERSION
```

Comment:
I think that we can keep this change, but I would prefer if our builds use 'stable'.